lib/src/mt_dev.c

```diff
@@ -1622,7 +1625,47 @@ static int dev_if_init_virtio_user(struct mt_interface* inf) {
   }
   inf->virtio_port_id = virtio_port_id;
 
-  info("%s(%d), succ\n", __func__, port);
+  sockfd = socket(AF_INET, SOCK_DGRAM, 0);
```

Comment:
can we move these code to a function under `mt_socket.c`?

Reply:
Sure.